src/sdk/actions.ts

```diff
@@ -99,12 +109,12 @@ export async function createActionsPlugin<TConfig = unknown, TEnv = unknown, TSu
     }
 
     if (pluginOptions.postCommentOnError && loggerError) {
-      await postComment(context, loggerError);
+      await postErrorComment(context, loggerError);
     }
   }
 }
 
-async function postComment(context: Context, error: LogReturn) {
+async function postErrorComment(context: Context, error: LogReturn) {
```

Comment:
isn't this function basically the same as `postComment`, only the github workflow run url is added so we could merge like
```
body: [message.logMessage.diff, `<!-- ${getGithubWorkflowRunUrl()} -->`, metadata].join("\n"),
```

Reply:
I'll have a look but I assume that `getGithubWorkflowRunUrl` wouldn't be available if we are within a worker.

Reply:
@whilefoo A thing we could do for Workers is to put the url to the logs, that would look like
`https://dash.cloudflare.com/${env.CLOUDFLARE_ACCOUNT_ID}/workers/services/view/${env.CLOUDFLARE_WORKER_NAME}/production/observability/logs?granularity=0&time=${timeParam}`

However this would require worker deployments to upload these values as they are not accessible through a worker by default. What do you think?

Reply:
Sure, we can make a new task

Reply:
https://github.com/ubiquity-os/ubiquity-os-kernel/issues/180